packages/backend/src/repositories/mailer.ts

```diff
@@ -0,0 +1,15 @@
+import { TransactionalEmailsApi, TransactionalEmailsApiApiKeys } from 'sib-api-v3-typescript';
+import { SendSmtpEmail as ISendSmtEmail } from 'sib-api-v3-typescript'
+import { getEnv } from '@helpers';
+
+export class MailerRepository {
```

Comment:
Create interface `IEmailProvider`
Create file `src/providers/email/SendInBlueEmailProvider.ts` and move all this to that file. implement IEmailProvider.

Reply:
We need interface to easily change provider (for example from SendInBlue to SendGrid or any other)